docs/usage.rst

```diff
@@ -206,11 +205,11 @@ prores.py
 makedip.py
 ~~~~~~~~~
 
--  Runs bitc.py or prores.py but only for use with accessioned packages.
--  Usage for running bitc.py all accessioned objects in a parent directory -
-   ``makedip.py path/to/lots_of_accessioned_directories -o path/to/output``
+-  Runs bitc.py or prores.py but only for use with AIPs.
+-  Usage for running bitc.py on all objects in a batch of information packages -
```

Comment:
Isn’t it specifically AIPs, and not sips or dips?

Reply:
There seems to be a contradiction here, probably inherited from my issues. It says that this can only be used with AIPs but it also says that it can be run on packages with OE numbers.

Reply:
@hyz2993 line 209 - suggest removing 'but only for use with AIPs.'

Reply:
Fixed.